Utils/github_workflow_scripts/close_stale_contrib_branches.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import urllib3
+import inspect
+from github import Github, enable_console_debug_logging
+from github.Repository import Repository
+from typing import List
+from dateparser import parse
+from datetime import datetime, timezone
+from demisto_sdk.commands.common.tools import run_command, print_error, print_warning
+
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+
+def get_stale_branch_names_with_contrib(repo: Repository) -> List[str]:  # noqa: E999
+    """Return the list of branches that have the prefix of "contrib/" without open pull requests
+    and that have not been updated for 2 months (stale)
+
+    Args:
+        repo (Repository): The repository whose branches will be searched and listed
+
+    Returns:
+        (List[str]): List of branch names that are stale and have the "contrib/" prefix
+    """
+    now = datetime.now(timezone.min)
+    branch_names = []
+    for branch in repo.get_branches():
+        if branch.name.startswith('contrib/'):
+            prs_with_branch_as_base = repo.get_pulls(state='OPEN', base=branch.name)
```

Comment:
make sure it is not used as the target (the internal PR)

Reply:
As @yaakovi said, you need to make sure there isn't an open internal PR for the branch.